bpf/lib/common.h

```diff
@@ -405,10 +405,10 @@ struct ct_state {
 	__u32 src_sec_id;
 };
 
-/* Lifetime of a proxy redirection entry is an entire day. All proxies should
+/* Lifetime of a proxy redirection entry is five minutes. All proxies should
  * be using TCP keepalive to force some traffic over the connection
- * periodically. */
-#define PROXY_DEFAULT_LIFETIME 86400
+ * periodically to keep these entries alive. */
+#define PROXY_DEFAULT_LIFETIME 360
```

Comment:
The keepalive time is 5 minutes as well. I'm fine with the above but then we have to reduce the keepalive timer in 
`pkg/proxy/dialer.go:func setKeepAlive(c net.Conn) error {`

Reply:
Thanks for the pointer. In the latest version of the patch I adjusted the `PROXY_DEFAULT_LIFETIME` to 720, since I saw some guides online suggesting TCP keepalives of around 10 minutes. Hopefully this is a more robust value for it, but still keeps a somewhat unusual interval that would hopefully become apparent in monitoring if something goes wrong (eg, drops every 12 minutes). 

I don't have a particularly methodic approach to determining these values so if anyone has contrary suggestions for these values then by all means, suggest them.